clientset/client.go

```diff
@@ -18,6 +18,7 @@ package v1
 
 import (
 	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/kubernetes"
```

Comment:
I might keep this file unchanged, and then import the kubernetes clientset in the runner client instead (see that file).

Reply:
Sounds good